src/test/java/org/opensearch/neuralsearch/stats/events/SemanticHighlightingStatsTests.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.neuralsearch.stats.events;
+
+import org.opensearch.test.OpenSearchTestCase;
+import org.junit.Before;
+
+import java.util.EnumSet;
+import java.util.Locale;
+
+public class SemanticHighlightingStatsTests extends OpenSearchTestCase {
```

Comment:
Do we need these tests? We're essentially checking if the enum we created exists within the enum we created.